code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -352,3 +352,14 @@
 				holding = null
 				. = TRUE
 	update_icon()
+	
+	/obj/machinery/portable_atmospherics/canister/proto
```

Comment:
I made a beautiful system that gives Engineering incentives without dumping shotguns/tesla guns/explosives/slave armies/powers into their lap. People wanted improved versions of current equipment and now they got it... sheesh.